app/javascript/api/invoices.js

```diff
@@ -56,4 +56,13 @@ export default {
       url: path,
     });
   },
+  notifyUser(user_name, user_id) {
+    const path = 'api/v1/slack';
+
+    return api({
+      method: 'post',
+      url: path,
+      data: {user_name, user_id},
```

Comment:
A space is required after '{'                   object-curly-spacing<br>Identifier 'user_name' is not in camel case     camelcase<br>Identifier 'user_id' is not in camel case       camelcase<br>A space is required before '}'                  object-curly-spacing